package.json

```diff
@@ -1,11 +1,12 @@
 {
   "version": "1.0.0-alpha.0",
-  "author": "Community for NL Design System",
+  "author": "Community for Logius Design System",
   "description": "Design system based on the NL Design System architecture",
   "license": "EUPL-1.2",
-  "name": "@example/design-system",
+  "name": "@lux/design-system",
   "keywords": [
```

Comment:
Zullen we voor de herkenbaarheid/vindbaarheid 'logius' ook toevoegen?

Reply:
Goeie. Toegevoegd